products/compute/terraform.yaml

```diff
@@ -1187,6 +1187,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       name: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateGCPName'
+  RegionNetworkEndpointGroup: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
you'll also want to add excludes to ansible.yaml and inspec.yaml for this resource